Source/WebCore/Modules/WebGPU/Implementation/WebGPUBufferImpl.cpp

```diff
@@ -83,14 +83,14 @@ void BufferImpl::getMappedRange(Size64 offset, std::optional<Size64> size, Funct
     callback({ static_cast<uint8_t*>(pointer) - actualOffset, actualSize });
 }
 
-auto BufferImpl::getBufferContents() -> MappedRange
+std::span<uint8_t> BufferImpl::getBufferContents()
 {
     if (!m_backing.get())
-        return { nullptr, 0 };
+        return { };
 
     auto* pointer = wgpuBufferGetBufferContents(m_backing.get());
     auto bufferSize = wgpuBufferGetCurrentSize(m_backing.get());
-    return { static_cast<uint8_t*>(pointer), static_cast<size_t>(bufferSize) };
+    return std::span { static_cast<uint8_t*>(pointer), static_cast<size_t>(bufferSize) };
```

Comment:
Why do we have to write `std::span` here?

Reply:
Likely not, will drop.